docs/source/models.rst

```diff
@@ -4271,4 +4271,11 @@ For more info, see:
     mlflow deployments list --help
     mlflow deployments get --help
     mlflow deployments run-local --help
-    mlflow deployments help --help
\ No newline at end of file
+    mlflow deployments help --help
+
+Community Model Flavors
+-----------------------
+
+Go to the `Community Model Flavors <https://mlflow.org/docs/latest/community-model-flavors.html#>`_
```

Comment:
The link binds to "latest" doc version. Could you modify the link to make it only include "community-model-flavors.html" part ?

Reply:
make sense, I just fixed it